templates/layouts/_questionnaire.html

```diff
@@ -37,4 +37,12 @@
           })
       }}
   {% endblock %}
+
+  {% if 'return_to_hub_url' in content %}
+      {% if content.return_to_hub_url != None %}
```

Comment:
The first `if` can be removed and the second `if` can just be:
```jinja2
  {% if content.return_to_hub_url %}
      <p class="u-mt-xl">{{ _("Can't complete this question?") }}<br>
      <a href="{{ content.return_to_hub_url }}">{{ _("Choose another section and return to this later") }}</a></p>
  {% endif %}
```

Reply:
Fixed, thanks!